### PR TITLE
[new release] passage (0.1.4)

### PR DESCRIPTION
**Pull request by thatportugueseguy**
Passage - used to store and manage access to shared secrets

- Project page: <a href="https://github.com/ahrefs/passage">https://github.com/ahrefs/passage</a>

##### CHANGES:

- Add bash completions on install

### CODE REVIEW

File: packages/passage/passage.0.1.4/opam [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/ahrefs/passage.git"
+install: [make "install_bash_completions"]
```
Comment (Khady) [inline packages/passage/passage.0.1.4/opam:43]: I think that this breaks installation of the binary. Please do not merge as is

**Comment by Khady** [comment]
> I think that this breaks installation of the binary. Please do not merge as is

long term suggestion for the CI, might be interesting to have a way to compare what is installed by a new release to what was installed by a previous one

**Comment by shonfeder** [comment]
> > I think that this breaks installation of the binary. Please do not merge as is
> 
> long term suggestion for the CI, might be interesting to have a way to compare what is installed by a new release to what was installed by a previous one

I think this would fall under https://github.com/ocurrent/opam-repo-ci/issues/309 and I'm updating it accordingly. Thanks :)

**Comment by thatportugueseguy** [comment]
closing as this isn't working as expected.

**Pull request closed without merging**